src/javascripts/helpers/data/authData.js

```diff
@@ -1 +1,21 @@
-import firebase from 'firebase/app'
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+const authDiv = $('#auth');
+const homeDiv = $('#homepage');
+const logoutButton = $('#navbar-logout-button');
+const checkLoginStatus = () => {
```

Comment:
give us a newline in between method definitions and variable definitions (just one right above this line is fine)